javascript/index.js

```diff
@@ -54,10 +54,21 @@ function setResetBtn() {
   // ... your code goes here
 }
 
+
+
+
 // Start/Stop Button
-btnLeft.addEventListener('click', () => {
-  // ... your code goes here
-});
+btnLeft.addEventListener('click', () => {  
+  if (btnLeft.classList.contains("btn start")) {
+    btnLeft.classList.add("btn stop");
+    btnRight.classList.add("btn split")
+    console.log("Running")
+  } else if (!btnLeft.classList.contains("btn start")) {
```

Comment:
The logic was good but there were some issues.
Instead of: !btnLeft.classList.contains("btn start")
Do btnLeft.classList.contains("stop")
Also, you are adding the new classes (stop, split, start, reset) but you are not removing the previous ones, so you won't be able to change back and forth between them.